py/legacypipe/catalog.py

```diff
@@ -3,29 +3,30 @@
 import numpy as np
 
 from tractor import PointSource, getParamTypeTree, RaDecPos
-from tractor.galaxy import ExpGalaxy, DevGalaxy, FixedCompositeGalaxy
+from tractor.galaxy import ExpGalaxy, DevGalaxy
+from tractor.sersic import SersicGalaxy
+from tractor.sersic import SersicGalaxy
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [redefinition of unused 'SersicGalaxy' from line 7](https://app.codacy.com/app/dstndstn/legacypipe/pullRequest?prid=4730106)